### PR TITLE
Adding ticket list with actions

### PR DESCRIPTION
**Pull request by ApaeP**

### CODE REVIEW

File: kitt_bar.3s.rb [hunk 1]
```diff
@@ -14,33 +14,10 @@
 
 BATCH_INFOS = [
   { slug: 1030, type: 'FT', cursus: 'Web', city: 'Paris'},
-  { slug: 1003, type: 'PT', cursus: 'Web', city: 'Paris'},
+  { slug: 984, type: 'FT', cursus: 'Web', city: 'Paris'},
 ]
 
 OLD_BATCHES = [
```
Comment (ApaeP) [inline kitt_bar.3s.rb]: This needs to be from user's config. Will make a branch to fix this 🤓

File: kitt_bar_app/config/http_kitt.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class HttpKitt
```
Comment (ApaeP) [inline kitt_bar_app/config/http_kitt.rb:1]: 🥳

File: kitt_bar_app/current_batch.rb [hunk 3]
```diff
@@ -40,14 +40,23 @@ def header
   end
 
   def tickets
-	  puts "- Tickets|href=#{tickets_url}|size=12"    
+	  puts "- Tickets|size=12"
+
+    @api_data.dig('tickets').each { |ticket|
+      puts "-- #{ticket.dig('user', 'name')} #{assigned_ticket(ticket)}"
+      ## TODO : two different puts for Take or Leave
```
Comment (ApaeP) [inline kitt_bar_app/current_batch.rb]: Is this a TODO added in a PR? 🤓

File: kitt_bar_app/current_batch.rb [hunk 4]
```diff
@@ -40,14 +40,23 @@ def header
...
+    }
   end
 
   def end_ticket
     return unless @ticket
 
-    url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
-    puts "- Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
+    # url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
+    puts "- Validate ticket with #{@ticket.dig('user','name')} | #{HttpKitt.put(@ticket, "done")}"
```
Comment (ApaeP) [inline kitt_bar_app/current_batch.rb:58]: Is this still relevant as it seems to be implemented in the `tickets` method above?

File: kitt_bar_app/current_batch.rb [hunk 5]
```diff
@@ -86,6 +95,11 @@ def user_ticket
     @api_data.dig('tickets')&.find { |ticket| ticket.dig('is_mine') }
   end
 
+  def assigned_ticket(ticket)
+    return if ticket.dig('assigned', 'id').nil?
```
Comment (ApaeP) [inline kitt_bar_app/current_batch.rb:98]: Style: leave an empty line after guard close #rubocopped 🤓

**Review by ApaeP — COMMENTED** [review]
Oh yeah 🥳 thank you!

File: kitt_bar_app/current_batch.rb (hunk 3 above)
Comment (paultursuru) [reply kitt_bar_app/current_batch.rb]: Ah forgot to remove the comment !

File: kitt_bar_app/current_batch.rb (hunk 4 above)
Comment (paultursuru) [reply kitt_bar_app/current_batch.rb:58]: I think it's nice to still have a quick "validate ticket" button without having to go down in the tickets menu > name > validate

File: kitt_bar_app/current_batch.rb (hunk 5 above)
Comment (paultursuru) [reply kitt_bar_app/current_batch.rb:98]: Will correct that !

File: kitt_bar_app/current_batch.rb (hunk 4 above)
Comment (ApaeP) [reply kitt_bar_app/current_batch.rb:58]: Indeed 🤓

**Pull request merged**